packages/server/src/utils/ldap.ts

```diff
@@ -0,0 +1,74 @@
+import { Client } from 'ldapts';
+import { User } from '../services/database/types';
+import Logger from '@joplin/utils/Logger';
+import { LdapConfig } from './types';
+
+export default async function ldapLogin(email: string, password: string, user: User, config: LdapConfig): Promise<User> {
+	const logger = Logger.create('LDAP');
```

Comment:
This should be created a the top (outside the function), otherwise it will create a new logger on each function call